api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -447,6 +447,11 @@ type NonAdmin struct {
 	// which restore spec field values to enforce
 	// +optional
 	EnforceRestoreSpec *velero.RestoreSpec `json:"enforceRestoreSpec,omitempty"`
+
```

Comment:
`s/non admin objects/non admin related objects`
We are cleaning up residual Velero Objects and secrets, right ? (Not the actual Non-Admin prefixed objects)

Reply:
yes, objects NAC controller created in OADP namespace and for some reason they were not properly deleted, GC will do this job

will make the change

Reply:
Please revise PR title/description of the same detail.

Reply:
do not see other places that needs changing. Can you point it to me?

Reply:
> feat: Garbage collector input

do not convey that this GC is limited to NAC created resources in oadp-operator namespace.